dist.ini

```diff
@@ -1,5 +1,7 @@
 name = Pod-TreeWalker
 
+source-url = https://github.com/raku-community-modules/Pod-TreeWalker.git
```

Comment:
OOC, is this a App::Mi6 feature I'm not aware of?

Reply:
Interesting.  Couldn't `App::Mi6` not deduce that from `.git/config` [remote "origin"] setting?